jax/numpy/lax_numpy.py

```diff
@@ -897,6 +897,46 @@ def histogram(a, bins=10, range=None, weights=None, density=None):
     counts = counts / bin_widths / counts.sum()
   return counts, bin_edges
 
+_range = range
+@_wraps(np.histogramdd)
+def histogramdd(sample, bins=10, range=None, weights=None, density=None):
+  sample = asarray(sample)
```

Comment:
Before calling `asarray(sample)`, call `_check_arraylike("histogramdd", sample)`. The reason for this is that we try to avoid silent conversion of lists to arrays, because it can be a silent source of poor performance.

Reply:
will do